docs/contents/user-docs/irsa.md

```diff
@@ -112,68 +113,43 @@ ACK_K8S_RELEASE_NAME=ack-$AWS_SERVICE_NAME-controller
 kubectl create namespace "$ACK_K8S_NAMESPACE"
```

Comment:
Once again, can be deleted. Add `--create-namespace` to `helm install` further down.